pyproject.toml

```diff
@@ -21,6 +21,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
+argcomplete = "*"
```

Comment:
I do like the idea of not having to vendor the package. I believe the original thought was not not have any dependencies other than recline itself, thus freeing it from needing to manage dependency versions, etc.

So the question is, is recline really compatible with every version of argcomplete? That seems unlikely, although figuring out which versions is tricky as well. Perhaps keeping a reasonable range here would be better. Say ">=3.0.0" which is their current major version and was released ~2 years ago.

Reply:
Fine for me. I didn't knew which version to put in here so I just added a wildcard.

Reply:
```suggestion
argcomplete = ">=3"
```